pkg/labels/selector_test.go

```diff
@@ -417,7 +417,7 @@ func TestRequirementConstructor(t *testing.T) {
 		{"x", ExistsOperator, nil, true},
 		{"1foo", InOperator, util.NewStringSet("bar"), true},
 		{"1234", InOperator, util.NewStringSet("bar"), true},
-		{strings.Repeat("a", 64), ExistsOperator, nil, false}, //breaks DNS rule that len(key) <= 63
+		{strings.Repeat("a", 64), ExistsOperator, nil, true}, //breaks DNS rule that len(key) <= 63
```

Comment:
should the comment be removed and have this case replaces with a strings.Repeat(254) ?